src/views/create-account-view.ts

```diff
@@ -3,11 +3,44 @@ import '../ga-heading';
 import '@material/mwc-textfield';
 import '@material/mwc-button';
 
-import { css, customElement, html, LitElement, TemplateResult } from 'lit-element';
+import { css, customElement, html, LitElement, property, TemplateResult } from 'lit-element';
 
 @customElement('create-account-view')
 export class CreateAccountView extends LitElement
 {
+    @property()
+    userName = '';
```

Comment:
Please use explicit `private` keywords here to make sure those fields are not unnecessarily exposed as public or protected API.

Also, those following properties all have no `@state` or `@property` decorators. They are needed for each single property. They do not work on blocks.